mangaki/mangaki/utils/mal.py

```diff
@@ -384,4 +451,24 @@ def import_mal(mal_username: str, mangaki_username: str):
                     score=user_work.score).save()
                 fails.append(user_work.title)
 
-    return nb_added, fails
+    existing_ratings = (
+        Rating.objects.filter(user=user, work__in=work_ids_to_import)
+            .values_list('work')
+            .all()
+    )
+
+    for existing_rating in existing_ratings:
+        work_ids_to_import.remove(existing_rating[0])
+
+    ratings = []
+    for work_id in work_ids_to_import:
```

Comment:
I believe `work_ids_to_import` can be removed and `scores` (which could be renamed to `works_to_import`) can be used instead, since `work_ids_to_import` is always `scores.keys()` (if we did `scores.pop(existing_rating[0], None)` at line 461).